src/app/jobr/Taschenrechner.java

```diff
@@ -0,0 +1,97 @@
+package app.jobr;
+
+import java.math.BigDecimal;
+
+public class Taschenrechner
+{
+    /**
+     * Does a simple mathematical calculation using the BigDecimal number format to prevent floating point inaccuracy.
+     * Supported: +, -, *, /, %. Result is printed in the command line.
+     *
+     * @param args The arguments given to the calculator. Expected format: number, operator, number
+     */
+    private BigDecimal zahl1;
+	private BigDecimal zahl2;
+	private BigDecimal result;
+	private String operation;
+
+    public static void main(String[] args)
```

Comment:
Ich würde die Main Methode hier aus der Taschenrechnerklasse herauslassen.
In den Unit Tests hast du sicher schon gemerkt, dass das nicht notwendig ist und so ist die Klasse ein wenig sauberer